releases/7.6.0.md

```diff
@@ -312,17 +364,34 @@ Minor incompatibilities:
 
     # Only handle i18n routing in active project. This will still handle the routes provided by Arches core and Arches applications,
     # but handling i18n routes in multiple places causes application errors.
-    if settings.APP_NAME != "Arches" and settings.APP_NAME not in settings.ARCHES_APPLICATIONS:
+    if settings.APP_NAME == settings.INSTALLED_APPS[-1]:
         if settings.SHOW_LANGUAGE_SWITCH is True:
             urlpatterns = i18n_patterns(*urlpatterns)
 
         urlpatterns.append(path("i18n/", include("django.conf.urls.i18n")))
-        
     ```
 
-11. Run `python manage.py updateproject`
+11. Add an `apps.py` in the project root:
```

Comment:
Sure, no problem. The only reason I waited was because I wasn't sure what `verbose_name` to derive for an arbitrary name, but we can just do whatever Django does by default.